DigitalLearningSolutions.Web.Tests/Controllers/LearningPortal/CurrentTests.cs

```diff
@@ -80,21 +80,21 @@ public async Task Current_action_should_not_fetch_ActionPlanResources_if_signpos
         public async Task Current_action_should_fetch_ActionPlanResources_if_signposting_enabled()
         {
             // Given
-            GivenCurrentActivitesAreEmptyLists();
-            A.CallTo(() => config[ConfigHelper.UseSignposting]).Returns("false");
+            GivenCurrentActivitiesAreEmptyLists();
+            A.CallTo(() => config[ConfigHelper.UseSignposting]).Returns("true");

```

Comment:
I noticed these Current tests were messed up and not actually doing what the title suggested (I probably did some copying and pasting and forgot to change the test logic around). I have corrected them here.

Also corrected a typo in "GivenCurrent**Activites**AreEmptyLists" to "GivenCurrent**Activities**AreEmptyLists"